cmd/sidecar/main.go

```diff
@@ -21,58 +19,41 @@ var infoLogger = log.InfoLogger
 
 func main() {
 	var err error
-	var secretsHandler *secrets.Secrets
 
-	// Create new Storage
-	configStorage, err := storageConfig.NewFromEnv()
+	// Initialize configurations
+	authnConfig, err := authnConfigProvider.NewFromEnv()
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
 
-	storageHandler, err := storage.NewStorageHandler(*configStorage)
+	secretsConfig, err := secretsConfigProvider.NewFromEnv()
```

Comment:
I think we should move this logic inside NewStorageHandler function

Reply:
done. it may change in the future to one config and then we'll get it from main anyway.